tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -118,14 +120,20 @@ def _process_and_assert_code_mapping(
                 mock_incr.assert_any_call(
                     key=f"{METRIC_PREFIX}.code_mapping.created", tags=tags, sample_rate=1.0
                 )
+            if expected_in_app_stack_trace_rules is not None:
+                # XXX: Grab it from the option
+                assert expected_in_app_stack_trace_rules == in_app_stack_trace_rules
```

Comment:
Right now, we pass `expected_in_app_stack_trace_rules`, however, when we don't use dry_run mode we will need to check the project option.